cmd/utils/grpcclient/client.go

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT License was not distributed with this
+// file, you can obtain one at https://opensource.org/licenses/MIT.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+package grpcclient
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/grpc"
+)
+
+// grpcClient is a helper for establishing grpc client connection reusing
+// default RPCConfiguration struct. TODO: It seems similar impl already used in
+// wallet pkg and test harness. Refactor this by providing a single client.
```

Comment:
there is an ulterior implementation in the `pkg/rpc/client` for JWT-authenticated sessions. It is out of scope to unify these clients, but we need to review that sooner or later